lib/command-context.coffee

```diff
@@ -29,6 +30,12 @@ class CommandContext
       runtime.didNotBuildArgs errorSendByArgs
       return false
 
+    if not runOptions.workingDirectory? or runOptions.workingDirectory is ''
+      # Precondition: lang? and lang of grammarMap
+      commandContext.workingDirectory = grammarMap[codeContext.lang][codeContext.argType].workingDirectory or ''
```

Comment:
With the default as `''`, will it be the base path of the open project that is used?